views/default/resources/ckeditor/browse.php

```diff
@@ -14,31 +14,6 @@
 
 $dir = new \DirectoryIterator($ck_dir);
 
-foreach ($dir as $file) {
```

Comment:
It just processes the gif and png image here, now all image types have the same process, and so I deleted the codes.